apps/costcontrol/test/unit/startup_test.js

```diff
@@ -280,12 +279,36 @@ suite('Application Startup Modes Test Suite >', function() {
         assertFTEStarted(applicationMode, done);
 
         // Launch the second start-up
-        MockNavigatorMozIccManager.triggerEventListeners('iccdetected', {});
+        var eventDetail = { detail: {serviceId: 'data'}};
+        var airplaneModeDisabledEvent =
+          new CustomEvent('airplaneModeDisabled', eventDetail);
+        window.dispatchEvent(airplaneModeDisabledEvent);
       });
 
       CostControlApp.init();
   });
 
+  test('SIM is not detected, the icc request is failing',
+    function(done) {
+      loadBodyHTML('/index.html');
+
+      this.sinon.stub(SimManager, 'requestDataSimIcc',
+                      failingRequestDataSIMIccId);
+
+        // Config the app to start (FTE)
+        var applicationMode = 'DATA_USAGE_ONLY';
+        window.CostControl = new MockCostControl();
+        window.ConfigManager = new MockConfigManager({
+          applicationMode: applicationMode
+        });
+
+      // airplanemode is disabled
```

Comment:
Remove the comment. Indeed, if this is the default, remove the code line as well.